app/scripts/modules/core/src/pipeline/config/stages/evaluateVariables/EvaluateVariablesStageConfig.tsx

```diff
@@ -0,0 +1,75 @@
+import { IStageConfigProps } from 'core/pipeline/config/stages/core/IStageConfigProps';
```

Comment:
Imports order

Reply:
Nice catch, thanks. Copied the wrong order from `WaitStageConfig` without noticing!

I'll fix the order there as well in the aforementioned `other PR`.

Reply:
I learned by watching @jrsquared